src/LTaaS/DomainClient.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace UKFast\SDK\LTaaS;
+
+use UKFast\SDK\Page;
+use UKFast\SDK\LTaaS\Entities\Domain;
+
+class DomainClient extends Client
+{
+    protected $basePath = 'ltaas/';
+
+    /**
+     * Gets paginated response for all of the domains
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getPage($page = 1, $perPage = 15, $filters = [])
+    {
+        $page = $this->paginatedRequest('v1/domains', $page, $perPage, $filters);
+        $page->serializeWith(function ($item) {
+            return new Domain($item);
+        });
+
+        return $page;
+    }
+
+    public function getDomain($domainId)
```

Comment:
please can you rename this to `getById` to be consistent with other service clients

Reply:
Changed